src/textual/geometry.py

```diff
@@ -218,6 +219,8 @@ def contains_point(self, point: tuple[int, int]) -> bool:
     def __contains__(self, other: Any) -> bool:
         try:
             x, y = other
+            assert isinstance(x, int)
```

Comment:
I consider this a bug fix since the message says they need to be ints, but there is no guarantee that they are. Not only that, if they are floats the function may even return True, confusing the enemy (the programmer, that is).

Reply:
I wouldn't consider the lack of a check to be a bug per se. Not averse to a sanity check, but it is an expensive one. Not sure it is worth the effort.

I'd also avoid raising an assert, just to catch it on the next line.

Reply:
True, the check here is a non-trivial overhead over the rest of the code. And changing the prototype to accept `tuple[int, int]` is an LSP violation. I'll remove this, unless I can prove that the whole function is redundant and can be removed (i.e. that all callsites use `contains_point()`).